build.R

```diff
@@ -211,6 +243,16 @@ plots <- list(
       displayName = "t-statistics",
       plotType = c("multiFeature", "multiTest"),
       packages = c("gplots", "viridis")
+    ),
+    single_feature_plotly = list(
+      displayName = "Expression by cell type interactive with plotly",
+      plotType = "singleFeature",
```

Comment:
The `plotType` vector also needs to include `"plotly"` so that the app (and `plotStudy()`) know that the fucntion creates a dynamic plotly graph.